manifests/mpm.pp

```diff
@@ -120,10 +137,8 @@
         }
       }
 
-      if versioncmp($apache_version, '2.4') < 0 {
```

Comment:
hmm.. this change wasn't SLES specific...are we changing debian behavior here on purpose?